### PR TITLE
E2E tests in local runner

### PR DESCRIPTION
**Pull request by mvrshvl**

### CODE REVIEW

**Review by SayMurZippowich — APPROVED** [review]
Cool thing!

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -24,89 +23,3 @@ jobs:
...
-#      - run: make build
-#      - run: |
-#          cd $GOPATH/src/github.com/youricorocks
-#          git clone https://github.com/youricorocks/avapegzone
-#          cd avapegzone && git checkout $AVAPEGZONE_COMMIT
-#          make build-ci build-faucet-ci
-#          cp -a $GOPATH/src/github.com/youricorocks/avalanchego/build/. ./build/
-#      - run: cd $GOPATH/src/github.com/youricorocks/avalanchego && make docker-build-ci
-#      - run: cd $GOPATH/src/github.com/youricorocks/avapegzone && make docker-build-ci
-#      - run: |
```
Comment (EldarKurbanov) [inline .github/workflows/ci.yml:56]: Are we really need this comments?

Comment (mvrshvl) [reply .github/workflows/ci.yml:56]: These comments are irrelevant. After connecting the server, the file for local launch will move here completely

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -10,8 +9,8 @@ on:
 
 env:
   LINT_VER: v1.41.1
-  AVAPEGZONE_COMMIT: master
-  AVARELAYER_COMMIT: master
+  AVAPEGZONE_COMMIT: main
```
[the remark this answers is not in the document]
Comment (mvrshvl) [reply .github/workflows/ci.yml:12]: Да

File: .github/workflows/ci.yml (hunk 1 above)
Comment (EldarKurbanov) [reply .github/workflows/ci.yml:56]: Whoops my bad

**Pull request merged**